### PR TITLE
Update storage portion of DG and replace old images

### LINKED ISSUE #207: [DOC] Clean Up Design and Implementation Artefact from AB3 DG Storage

### PR DESCRIPTION
**Pull request by yyhangz**
### Summary

- Resolves #207
- Change `delete t/doctor` and `list t/doctor` images in the UG tutorial so that it doesn't show highlighted rows

### Detail

- Change formatting of one section in UG to html formatting

_Looks like this initially, supposed to be an unordered list. 
Not sure why it looks like this, so I try to change it to html to see if it works better_
image

### Checks

- [X] Link PR to issue
- [X] Add Reviewer
- [x] Pass CI

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -940,8 +940,11 @@ Parameters      |  Explanation
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Note:**
```
Comment (simonjulianl) [inline docs/UserGuide.md:941]: oh the error because if wanna use markdown, use markdown="block" not span

**Review by simonjulianl — CHANGES_REQUESTED** [review]
just the markdown

**Review by simonjulianl — COMMENTED** [review]
Yup I check the diagram, no error there and should be fine

File: docs/UserGuide.md (hunk 1 above)
Comment (yyhangz) [reply docs/UserGuide.md:941]: ahh right ok lemme update that

**Review by simonjulianl — APPROVED** [review]
LGTM

**Pull request merged**